From b067a3c282a90029179d388ea0db5219e2967f2c Mon Sep 17 00:00:00 2001
From: randalhsu <xxx@gmail.com>
Date: Sat, 6 Nov 2021 11:13:05 +0800
Subject: [PATCH 4/4] setCrossHair for mobile

---
 src/gui/pane-widget.ts | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/src/gui/pane-widget.ts b/src/gui/pane-widget.ts
index b32607cb..121a49a4 100644
--- a/src/gui/pane-widget.ts
+++ b/src/gui/pane-widget.ts
@@ -508,14 +508,10 @@ export class PaneWidget implements IDestroyable, MouseEventHandlers {
 			const x = xx as Coordinate;
 			const y = yy as Coordinate;
 
-			if (!mobileTouch) {
-				this._setCrosshairPositionNoFire(x, y);
-			}
+			this._setCrosshairPositionNoFire(x, y);
 		} else {
 			this._state.model().setHoveredSource(null);
-			if (!isMobile) {
-				this._clearCrosshairPosition();
-			}
+			this._clearCrosshairPosition();
 		}
 	}
 
-- 
2.32.0 (Apple Git-132)

